.github/workflows/ci.yml

```diff
@@ -84,7 +89,8 @@ jobs:
 
       - name: "Run tests"
         run: |
-          docker run --rm -e SOFTHSM2_CONF="${{ matrix.softhsm2_conf }}" \
+          docker run --rm -e ASAN_OPTIONS="${{ env.ASAN_OPTIONS }}" -e LSAN_OPTIONS="${{ env.LSAN_OPTIONS }}" \
```

Comment:
What happens if ASAN options and LSAN options are provided but the build-type is a no-asan one?

I am not sure, if I like the overloading and somehow tend to create an extra step which is dependent on the build-type.

Reply:
They are ignored. Do you prefer if then else statements, with and without OPTION specification instead?

Reply:
yes. That would be great.
Then it is clear what is ran.

Reply:
I fixed this with matrix CI code instead, in order to avoid clunky if statements.